cmd/aws-sigv4-proxy/main.go

```diff
@@ -102,12 +102,14 @@ func main() {
 	} else {
 		credentials = session.Config.Credentials
 	}
-
+	
+	fmt.Printf("%t", useUnsignedPayload())
```

Comment:
Is this print needed? Seems like it was used for debugging?

Reply:
Removed the print and committed new version. Please check.